lib/platform/gitlab/index.spec.ts

```diff
@@ -300,6 +300,39 @@ describe(getName(), () => {
       });
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
+    it('should fall back respecting when GITLAB_IGNORE_REPO_URL is set', async () => {
+      const gitlabIgnoreRepoUrlBefore = process.env.GITLAB_IGNORE_REPO_URL;
+      process.env.GITLAB_IGNORE_REPO_URL = 'true';
+      const selfHostedUrl = 'http://mycompany.com';
```

Comment:
```suggestion
      const selfHostedUrl = 'http://mycompany.com/gitlab';
```

Just url visibillity, maybe better to add `/api/v4` here too.